src/main/java/seedu/duke/Eventus.java

```diff
@@ -38,6 +43,9 @@ public void run() {
                 Command c = Parser.parse(input);
                 if (c.getCommandType().equals("list venues")){
```

Comment:
perhaps a space can be added between `)` and `{` for the codestyle to be consistent throughout